main.go

```diff
@@ -49,6 +49,8 @@ func main() {
 		err = actions.FinalizeDocs()
 	case environment.ActionRelease:
 		err = actions.Release()
+	case environment.ActionLog:
+		err = actions.LogActionResult()
```

Comment:
This shouldn't return an error (we don't want to fail anything if we can't send logs) this should be best effort

Reply:
The only time it returns an error is if API key is not set. Any GH Action command would already have failed.

Reply:
I'm fine removing that though it doesn't really matter